keyboards/genovation_plum_92_teensy_lc/flash.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+./teensy_loader_cli --mcu=TEENSYLC -wv ../../genovation_plum_92_teensy_lc_default.hex
```

Comment:
Yes, the `teensy` target should be moved from `avr.mk` to `common.mk`

And then you make it with `make teensy`